gulpfile.babel.js

```diff
@@ -0,0 +1,105 @@
+import gulp from 'gulp';
+import util from 'gulp-util';
+import minimist from 'minimist';
+import rename from 'gulp-rename';
+import replace from 'gulp-replace';
+import path from 'path';
+import del from 'del';
+import coffee from 'gulp-coffee';
+import stream from 'vinyl-source-stream';
+import browserify from 'browserify';
+import coffeeify from 'coffeeify';
+import shim from 'browserify-shim';
+import { Server as Karma } from 'karma';
+
+import { version } from './package';
+import { standalone } from './package';
+import { filename } from './package';
+
+
+const source = './src/brainstem.coffee';
+const options = minimist(process.argv.slice(2));
+
+const moduleOutput = './lib';
+const gemOutput = './vendor/assets/javascripts';
+
+
+// Tasks
+
+gulp.task('build-module', () => {
+  return gulp.src('./src/**/*.coffee')
+    .pipe(coffee())
+    .pipe(gulp.dest(moduleOutput))
+    .on('error', util.log);
+});
+
+gulp.task('build-gem', () => {
```

Comment:
Do we actually want to be explicit about specifying parens when we're not providing arguments? Pretty sure there's an ESLint option we could leverage and I'd advocate for using it to enforce whatever convention we go with.

I think it would make sense to leave them out and have that enforced by the linter, but am fine with being explicit as long as we're consistent with what we're doing.